<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>9</y>
      <width>601</width>
      <height>511</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>561</width>
       <height>491</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="repos">
      <attribute name="title">
       <string>Repositories</string>
      </attribute>
      <widget class="QListWidget" name="reposList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>531</width>
         <height>301</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="initBtn">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>20</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Initialize Repository</string>
       </property>
      </widget>
      <widget class="QPushButton" name="selectBtn">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>410</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Select Repository</string>
       </property>
      </widget>
      <widget class="QPushButton" name="removeRepoBtn">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>20</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove Repository</string>
       </property>
      </widget>
      <widget class="QPushButton" name="openBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Open Existing Repository</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="files">
      <attribute name="title">
       <string>Tracked Files</string>
      </attribute>
      <widget class="QTableWidget" name="statusList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>531</width>
         <height>301</height>
        </rect>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>200</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>200</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Files</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Status</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="addBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Add File</string>
       </property>
      </widget>
      <widget class="QPushButton" name="commitBtn">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>90</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Commit Changes</string>
       </property>
      </widget>
      <widget class="QPushButton" name="removeFileBtn">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>90</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove File</string>
       </property>
      </widget>
      <widget class="QLabel" name="repoName">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>361</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>26</pointsize>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>No Repository Selected</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="refreshBtn">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>90</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
